tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -9,7 +9,7 @@
 #include <string_view>
 
 using namespace std;
-
+using namespace std::string_view_literals;

```

Comment:
This is unnecessary because the UDLs are designed as:
```cpp
namespace std {
    inline namespace literals {
        inline namespace string_view_literals {
            // ... define operator"" sv() ...
```
Members of an `inline namespace` can be accessed as members of the outer namespace. Therefore:
* `using namespace std::string_view_literals;` makes *just* the `sv` literals available, nothing else. (This itself is a short form of `using namespace std::literals::string_view_literals;` because `literals` is `inline`.)
* `using namespace std::literals;` makes all of the literals available (`sv`, `i`, `ms`, etc.) but nothing else.
* `using namespace std;` makes all of the functions/classes in `std` available, plus the literals - which is the case in this test.